beat/heart.py

```diff
@@ -1713,16 +1758,16 @@ def get_fomosto_baseconfig(
             id='any_P',
             definition=definition))
 
-    if 'any_S' in waveforms:
+    if 'any_S' in waveforms: # always add any_S to have automatic adjustment of GF window
         tabulated_phases.append(gf.TPDef(
             id='any_S',
             definition='s,S,s\\,S\\'))
 
     # surface waves
-    if 'slowest' in waveforms:
```

Comment:
always adding slowest may become very expensive for teleseismic setups, wouldnt want to hardcode add it.
Why do you need it here?